spatialmath/geometry.go

```diff
@@ -141,6 +142,9 @@ func (config *GeometryConfig) ParseConfig() (Geometry, error) {
 
 // NewGeometryFromProto instantiates a new Geometry from a protobuf Geometry message.
 func NewGeometryFromProto(geometry *commonpb.Geometry) (Geometry, error) {
+	if geometry.Center == nil {
```

Comment:
I don't think it can break anything because if this case was ever being hit it would cause a panic. 

I'm not sure how this was ever coming up in the first place though.  The proto does not allow for this to be optional https://github.com/viamrobotics/api/blob/72d2b65e9040748257577fafe662557b4ea9604b/proto/viam/common/v1/common.proto#L100

I do see that the protobuf struct has this as a pointer, so it does seem like adding this makes sense?